misc/python/materialize/cli/ci_closed_issues_detect.py

```diff
@@ -24,7 +24,8 @@
 ISSUE_RE = re.compile(
     r"""
     ( TimelyDataflow/timely-dataflow\#(?P<timelydataflow>[0-9]+)
-    | ( materialize\# | materialize/issues/ | \# ) (?P<materialize>[0-9]+)
+    | ( cloud\# | cloud/issues/ ) (?P<cloud>[0-9]+)
```

Comment:
The former line 27 should not be replaced.

Reply:
Oops, that was my local debugging :D